### PR TITLE
feat: seedlot details page grid

### PR DESCRIPTION
**Pull request by iatdaitan**
# Description

Created the new seedlot details page grid structure. The grid is not fully responsive yet, as we don't have the final design for it still.

Fixes #[FSADT2-48](https://apps.nrs.gov.bc.ca/int/jira/browse/FSADT2-478)

## Type of change
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?
Manually tested by the developer.

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have already been accepted and merged

---
Thanks for the PR!
Any successful deployments (not always required) will be available below.
[Frontned](https://nr-spar-webapp-47-frontend.apps.silver.devops.gov.bc.ca/) available
Once merged, code will be promoted and handed off to following workflow run.

---
Thanks for the PR!

Any successful deployments (not always required) will be available below.
[Frontend](https://nr-spar-webapp-47-frontend.apps.silver.devops.gov.bc.ca/) available

Once merged, code will be promoted and handed off to following workflow run.
[Main Merge Workflow](https://github.com/bcgov/nr-spar-webapp/actions/workflows/merge-main.yml)

### CODE REVIEW

**Comment by DerekRoberts** [comment]
@iatdaitan You've pulled in the pipeline overhaul PR, so please keep this PR in draft mode.  It tells people this PR isn't ready to merge and keeps the CI workflow from firing, which saves resources.

File: frontend/src/App.tsx [hunk 1]
```diff
@@ -48,6 +49,15 @@ const App: React.FC = () => {
               </ProtectedRoute>
             )}
           />
+
+          <Route
+            path="/seedlot-details"
```
Comment (mgaseta) [inline frontend/src/App.tsx:54]: Change the path here, 'seedlot' should be a starting point for the rest of the URL, like 'seedlot/detail'.

File: frontend/src/views/SeedlotDetails/index.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+const SeedlotDetails = () => (
+  <FlexGrid className="seedlot-details-page">
+    <Stack gap={7}>
+      <Row>
+        <PageTitle
+          title="Seedlot 636465"
+          subtitle="Check and manage this seedlot"
+          favourite
+        />
+      </Row>
```
Comment (mgaseta) [inline frontend/src/views/SeedlotDetails/index.tsx:27]: Missing 'Manage Seedlot' button.

File: frontend/src/views/SeedlotDetails/styles.scss [hunk 3]
```diff
@@ -0,0 +1,16 @@
+@use '../../theme/variables.scss' as vars;
+@use '../../theme/colors.scss' as colors;
+@use '@carbon/type';
```
Comment (mgaseta) [inline frontend/src/views/SeedlotDetails/styles.scss:3]: Remove unused imports.

File: frontend/src/views/SeedlotDetails/index.tsx [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+              </TabPanels>
+            </Tabs>
+          </Column>
+        </Row>
+      </section>
+    </Stack>
+  </FlexGrid>
+);
+
+export default SeedlotDetails;
```
Comment (mgaseta) [inline frontend/src/views/SeedlotDetails/index.tsx:57]: Add new line at the end of the file.

**Review by mgaseta — COMMENTED** [review]
Igor is on vacation, I'm assuming this PR and added markups for references when fixing it.

**Review by RMCampos — APPROVED** [review]
LGTM

**Review by lucasbuenoencora — APPROVED** [review]
LGTM!  :rocket:

**Pull request merged**